patch/core/book.js

```diff
@@ -26,19 +23,20 @@ module.exports = {
 			} else if (item.includes('crystal')) {
 				item = 'crystal';
 			}
-
+			
```

Comment:
Remove this unnecessary whitespace. I typically set the "remove trailing whitespace" preference in my editor's settings, not sure which one you're using.